test/fixture/main-nocolor.stdio

```diff
@@ -5,7 +5,6 @@ ACCPTST:STDIO_REPLACE_DATETIMES
 {datetime} UTC [     info] __dir: {root}
 {datetime} UTC [     info] __base: main
 {datetime} UTC [     info] OSTYPE: {OSTYPE}
-{datetime} UTC [     info] Script called directly? yes
```

Comment:
it looks like yarn is still broken, and without mentioning lanyon explicitly in package.json `npm ...` fails epically... I put it in plain old dependencies, but it looks like @kvz puts them in dev dependencies... https://github.com/yarnpkg/yarn/issues/760 is still unresolved, it appears